LINK/usr/bin/appliance_console

```diff
@@ -1,7 +1,10 @@
-#!/bin/bash
-# description: ManageIQ appliance console launcher
-#
+#!/usr/bin/env ruby
 
-[[ -s /etc/default/evm ]] && source /etc/default/evm
+require "bundler"
+Bundler.setup
 
-ruby /var/www/miq/vmdb/gems/pending/appliance_console.rb $@
+require "pathname"
+RAILS_ROOT = Pathname.new("/var/www/miq/vmdb")
+
+require "manageiq-gems-pending"
+require "appliance_console"
```

Comment:
Opened https://github.com/ManageIQ/manageiq-gems-pending/issues/12 to track moving the console away from being a script.